doc/schemas/WRITING_SCHEMAS.md

```diff
@@ -39,6 +39,9 @@ You should always list all fields which are *always* present in
 We extend the basic types; see
 [contrib/pyln-testing/pyln/testing/fixtures.py](fixtures.py).
 
+In addition, before to commit a new schema or a new version of it, make sure that it
```

Comment:
grammar nit:

In addition, before *committing* a new schema or a new version of it, make sure that it

Reply:
Thanks 😄